PterodactylLib/Classes/Pterodactyl.swift

```diff
@@ -33,29 +33,39 @@ public class Pterodactyl {
     }
     
     public func triggerSimulatorNotification(withFullPayload payload: [String: Any]) {
-        let endpoint = "http://\(host):\(port)/simulatorPush"
-        
+        let endpoint = "http://\(host):\(port)/\(pushEndpoint)"
```

Comment:
As I add endpoints, it makes it a bit easier to expose them if they aren't hard coded here.